ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -435,11 +408,8 @@ else
 	#
 	. ${R}etc/init.d/rc.serial
 
-	if [ $IO_PRESENT = no ]
```

Comment:
Is this correct with an IO?

Reply:
Yes, but we'll want to do more here in a follow up.
 - most boards with a px4io don't include drivers/rc_input, but should as an optional parameter configurable option (CRSF on arbitrary FMU serial port)
 - on a pixhawk 4 this will run and publish a 2nd input_rc instance that we don't do anything (kind of odd, but harmless)